includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -119,7 +119,10 @@ public function __construct(
 			'account_statement_descriptor' => [
 				'type'        => 'account_statement_descriptor',
 				'title'       => __( 'Customer bank statement', 'woocommerce-payments' ),
-				'description' => __( 'Edit the way your store name appears on your customers’ bank statements.', 'woocommerce-payments' ),
+				'description' => WC_Payments_Utils::esc_interpolated_html(
+					__( 'Edit the way your store name appears on your customers’ bank statements (read more about requirements <a>here</a>).', 'woocommerce-payments' ),
+					[ 'a' => '<a href="https://docs.woocommerce.com/document/payments/bank-statement-descriptor/">' ]
```

Comment:
Should this open in a new tab?

```suggestion
					[ 'a' => '<a href="https://docs.woocommerce.com/document/payments/bank-statement-descriptor/" target="_blank" rel="noopener noreferrer">' ]
```

Reply:
I'd say we should, thank you for the pointer.